src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/ProductSelectionByProducts.tsx

```diff
@@ -98,3 +99,28 @@ export function ProductSelectionByProducts({
     </View>
   );
 }
+
+/**
+ * This function will return the product name to be displayed.
+ *
+ * If the product name has alias set, then show the alias,
+ * otherwise get the regular product name.
+ *
+ * @param fareProduct product to be displayed
+ * @param language current language of the app
+ * @returns product name to be displayed, string
+ */
+function getProductName(
+  fareProduct: PreassignedFareProduct,
+  language: Language,
+): string {
+  const aliasText = fareProduct.productAlias
+    ? getTextForLanguage(fareProduct.productAlias, language)
+    : undefined;
+
+  const productName = aliasText
+    ? aliasText
+    : getReferenceDataName(fareProduct, language);
```

Comment:
I think we can simplify this - and at that point it's maybe not necessary with the helper function?

```suggestion
  const productName = fareProduct.productAlias
    ? getTextForLanguage(fareProduct.productAlias, language)
    : getReferenceDataName(fareProduct, language);
```

Reply:
Riiight, thank you!

Reply:
updated, but since the `getTextForLanguage` can return undefined, I modified it a bit.